presto-cli/src/main/java/com/facebook/presto/cli/ErrorMessages.java

```diff
@@ -23,22 +23,75 @@
 import com.google.common.collect.ImmutableList;
 import org.fusesource.jansi.Ansi;
 
+import java.io.EOFException;
 import java.io.PrintWriter;
 import java.io.StringWriter;
+import java.net.ConnectException;
 import java.util.List;
 
 import static com.facebook.presto.cli.ConsolePrinter.REAL_TERMINAL;
 import static com.google.common.base.Preconditions.checkState;
+import static com.google.common.base.Throwables.getCausalChain;
 import static java.lang.String.format;
 
 public class ErrorMessages
 {
+    private static final String PRESTO_COORDINATOR_NOT_FOUND = "There was a problem with a response from Presto Coordinator.\n";
+
     private static final String TECHNICAL_DETAILS_HEADER = "\n=========   TECHNICAL DETAILS   =========\n";
     private static final String SESSION_INTRO = "[ Session information ]\n";
     private static final String ERROR_MESSAGE_INTRO = "[ Error message ]\n";
     private static final String STACKTRACE_INTRO = "[ Stack trace ]\n";
     private static final String TECHNICAL_DETAILS_END = "========= TECHNICAL DETAILS END =========\n\n";
 
+    private enum Tip {
+        VERIFY_PRESTO_RUNNING("Verify that Presto is running on %s."),
+        DEFINE_SERVER_AS_CLI_PARAM("Use '--server' argument when starting Presto CLI to define server host and port."),
```

Comment:
I understand that concern.
Isn't `--quiet` mode (introduced later in this PR) an answer for you? In quiet mode only the main message is displayed and all tips are hidden. Wouldn't adding that flag to CLI while starting it through your wrapper be a solution to that?